index.bs

```diff
@@ -840,6 +840,17 @@ readable stream is <a>locked to a reader</a>.
   1. Return *true*.
 </emu-alg>
 
+<h4 id="is-readable-stream-readable" aoid="IsReadableStreamReadable" nothrow>IsReadableStreamReadable (
```

Comment:
This doesn't appear to be used in the spec, but only in the reference implementation.

Also, it's kind of a lame abstract operation? Is the idea just to preserve encapsulation in some way? Meh....

Reply:
After reading your comment, I realised that pipeTo() looks at the internals of WritableStream directly, so abstraction here probably isn't appropriate.